desktop/apps/consignments2/index.js

```diff
@@ -21,6 +21,12 @@ app.get(landing.paths.show, adminOnly, routes.landingPage)
 app.get(landing.paths.show + '/data', adminOnly, data)
 app.get(landing.paths.edit, adminOnly, edit)
 app.post(landing.paths.edit, adminOnly, upload)
-app.get('/consign2/submission', adminOnly, routes.submissionFlow)
-app.get('/consign2/submission/:id/thank_you', adminOnly, routes.submissionThankYou)
-app.get('/consign2/submission/:id/upload', adminOnly, routes.submissionUpload)
+app.get('/consign2/submission/:id/describe-your-work', routes.submissionFlowWithFetch)
+app.get('/consign2/submission/:id/upload-photos', routes.submissionFlowWithFetch)
+app.get('/consign2/submission/:id/thank-you', adminOnly, routes.submissionFlowWithId)
+app.get('/consign2/submission/:id/upload', adminOnly, routes.submissionFlowWithId)
+app.get('/consign2/submission', routes.submissionFlow)
+app.get('/consign2/submission/create-account', routes.redirectToSubmissionFlow)
```

Comment:
Just curious, is it possible to wildcard this [similar to prediction](https://github.com/artsy/prediction/blob/master/src/server.js#L153)? So you don't need to duplicate route-sets on the client and the server?